### ISSUE TITLE
closedByUser is not set in DB if the conversation is closed at the beginning

### ISSUE THREAD

**Issue by fulldecent** [freescout-helpdesk/freescout#1477]
I purchased the API/Webhooks module.

# Test case

1. Support agent logs in
2. Support agent clicks conversation button
3. In this conversation the status is marked as closed
4. Click send

# Expected outcome

The  `closedByUser` in API is set to that person.

# Actual outcome

`closedByUser` is null.

**Comment by freescout-helpdesk** [comment]
We can't reproduce the issue. Could you please attach screenshots or video of steps 1-4 and API output.

**Comment by fulldecent** [comment]
Yes.

I have just reproduced it on demo website.

<img width="1523" alt="Screen Shot 2021-10-13 at 16 02 00" src="https://user-images.githubusercontent.com/382183/137204620-1d346278-37d7-42b8-8027-8d3bbcf3a5ac.png">
<img width="1523" alt="Screen Shot 2021-10-13 at 16 02 02" src="https://user-images.githubusercontent.com/382183/137204628-abff2f0a-44d4-4ed5-850f-0bc73fa25d98.png">

---

Perhaps the error is here:

https://github.com/freescout-helpdesk/freescout/blob/078e098b28266ee5b7f2cd775a5b4e5aad6df48e/app/Conversation.php#L1881

It is missing something like this:

https://github.com/freescout-helpdesk/freescout/blob/078e098b28266ee5b7f2cd775a5b4e5aad6df48e/app/Conversation.php#L559

**Comment by freescout-helpdesk** [comment]
We'll leave it as `help wanted`. Maybe someone will be able to reproduce it and find the reason of the issue.

**Comment by fulldecent** [comment]
I have reproduced this on demo just now. Status is closed but closed by is null.

Did it twice in a row and got different results.

Closing this issue. If somebody is better at testing than me they can reopen it!

<img width="1310" alt="Screen Shot 2022-10-12 at 14 06 15" src="https://user-images.githubusercontent.com/382183/195416139-d01511e6-6098-4390-8dd6-2ee53fac2ec1.png">

<img width="1310" alt="Screen Shot 2022-10-12 at 14 06 07" src="https://user-images.githubusercontent.com/382183/195416134-da45ca8b-c068-491d-a7f5-eb72242d2d32.png">
